src/machine/board_circuitplay_express.go

```diff
@@ -70,10 +70,10 @@ const (
 
 // UART1 on the Circuit Playground Express.
 var (
-	UART1 = UART{Bus: sam.SERCOM1_USART,
+	UART1 = UART{
 		Buffer: NewRingBuffer(),
-		Mode:   PinSERCOM,
-		IRQVal: sam.IRQ_SERCOM1,
+		Bus:    sam.SERCOM4_USART,
+		SERCOM: 4,
```

Comment:
The pin configuration here (`PB08` / `PB09`) was never supported and wouldn't work on SERCOM1 anyway. Switched to SERCOM4 keeping the pin numbers, following the common pinout of the Circuit Playground Express.